### ISSUE TITLE
Keyboard support on tabs is broken

### ISSUE THREAD

**Issue by jnurthen** [adobe/spectrum-web-components#466]
### Expected Behaviour
Should be able to use arrow keys to move between tabs in a tab list

### Actual Behaviour
Arrow keys move to illogical tab pages

#### Steps to Reproduce

- https://opensource.adobe.com/spectrum-web-components/components/tab-list
- Press tab 4 times in the page to place focus on the "Tab 1" tab under the Example header
- Press right arrow
- Note focus does not move to Tab 2
- Press left arrow
- Note focus moves to Tab 3

#### Platform and Version
MacOS 
Chrome 79.x

**Comment by Westbrook** [comment]
Thanks for the issue @jnurthen!

I'm hoping this is a context issue rather than a pure functionality issue. Can you take a visit to https://opensource.adobe.com/spectrum-web-components/storybook/?path=/story/tabs--default and see if the experience there is more what you'd expect from a tabs element?

If so, I already see a path to a fix! Looking for some tweaks our testing/tooling to prevent it in the future.

If not, I'll look deeper.

**Comment by jnurthen** [comment]
Seems fine in storybook.
